recipes/sdl2/bld.bat

```diff
@@ -0,0 +1,40 @@
+:: MSVC library for Python 2.7 is not found automaticaly if it is 
+:: installed in the user's AppData folder. Add it to PATH
+setlocal EnableDelayedExpansion
+
+set TARGET=Release
+:: Set target back to Debug for 64-bits builds with VS2008 and VS2010
+if %VS_MAJOR% LSS 14 (
+	if %ARCH% == 64 (
+		set TARGET=Debug

```

Comment:
That `memset` error smells like the release build removing the CRT intrinsics to me. May have something to do with all this business about not being reliant on any CRT version.